e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+create_infra "$namespace"
+cluster="some-name"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running "$cluster-rs0" 3
+
+check_service() {
+	kubectl_bin patch psmdb "$cluster" --type=json --patch '[
+        {
+            "op": "replace",
+            "path": "/spec/ignoreAnnotations",
+            "value": ["ignoredAnnotation"]
+        },
+        {
+            "op": "replace",
+            "path": "/spec/ignoreLabels",
+            "value": ["ignoredLabel"]
+        }
+    ]'
+
+	sleep 10
+
+	replset_name=$1
+
+	local expose_path
+	local svc_name
+	case $replset_name in
+
+		"rs0")
+			expose_path="/spec/replsets/0/expose"
+			svc_name="$cluster-rs0-0"
+			;;
+
+		"cfg")
+			expose_path="/spec/sharding/configsvrReplSet/expose"
+			svc_name="$cluster-cfg-0"
+			;;
+
+		"mongos")
+			expose_path="/spec/sharding/mongos/expose"
+			svc_name="$cluster-mongos-0"
+			;;
+
+		*)
+			desc 'unknown check_service argument'
+			exit 1
+			;;
+	esac
+
+	desc "add labels and annotation manually in service $svc_name"
+	# `notIgnoredLabel` and `notIgnoredAnnotation` should be deleted
+	kubectl_bin patch "service/$svc_name" --type=json --patch '[
+        {
```

Comment:
For other operator it works otherwise. 
Until we add annotations/labels to cr operator shouldn't control custom annotations/labels.

Reply:
We have custom labels in the cr, so operator controls them